pkg/core/native/native_test/neo_test.go

```diff
@@ -117,6 +117,60 @@ func TestNEO_CandidateEvents(t *testing.T) {
 	aer = e.GetTxExecResult(t, tx)
 	require.Equal(t, 0, len(aer.Events))
 }
+func TestNEO_CommitteeEvents(t *testing.T) {
+	neoCommitteeInvoker := newNeoCommitteeClient(t, 100_0000_0000)
+	neoValidatorsInvoker := neoCommitteeInvoker.WithSigners(neoCommitteeInvoker.Validator)
+	e := neoCommitteeInvoker.Executor
+
+	cfg := e.Chain.GetConfig()
+	committeeSize := cfg.GetCommitteeSize(0)
+	validatorsCount := cfg.GetNumOfCNs(0)
+	freq := validatorsCount + committeeSize
+	advanceChain := func(t *testing.T) {
+		for i := 0; i < freq; i++ {
+			neoCommitteeInvoker.AddNewBlock(t)
+		}
+	}
+	voters := make([]neotest.Signer, committeeSize+1)
+	candidates := make([]neotest.Signer, committeeSize+1)
+	for i := 0; i < committeeSize+1; i++ {
+		voters[i] = e.NewAccount(t, 10_0000_0000)
+		candidates[i] = e.NewAccount(t, 2000_0000_0000) // enough for one registration
+	}
+	txes := make([]*transaction.Transaction, 0, committeeSize*4-2)
+	for i := 0; i < committeeSize+1; i++ {
+		transferTx := neoValidatorsInvoker.PrepareInvoke(t, "transfer", e.Validator.ScriptHash(), voters[i].(neotest.SingleSigner).Account().PrivateKey().GetScriptHash(), int64(committeeSize+1-i)*1000000, nil)
+		txes = append(txes, transferTx)
+		if i > 0 {
+			registerTx := neoValidatorsInvoker.WithSigners(candidates[i]).PrepareInvoke(t, "registerCandidate", candidates[i].(neotest.SingleSigner).Account().PublicKey().Bytes())
+			txes = append(txes, registerTx)
+			voteTx := neoValidatorsInvoker.WithSigners(voters[i]).PrepareInvoke(t, "vote", voters[i].(neotest.SingleSigner).Account().PrivateKey().GetScriptHash(), candidates[i].(neotest.SingleSigner).Account().PublicKey().Bytes())
+			txes = append(txes, voteTx)
+		}
+	}
+	//for _, tx := range txes {
+	//	e.CheckHalt(t, tx.Hash(), stackitem.Make(true))
+	//}
+
+	prevCommitteePublicKey, err := keys.NewPublicKeysFromStrings(cfg.StandbyCommittee)
+	require.NoError(t, err)
+	fmt.Println(prevCommitteePublicKey)
+
+	advanceChain(t)
+	newCommitteePublicKeys := make([]*keys.PublicKey, committeeSize)
+	for i, member := range neoCommitteeInvoker.Signers {
+		newCommitteePublicKeys[i] = member.(neotest.SingleSigner).Account().PublicKey()
+	}
+	tx := neoCommitteeInvoker.WithSigners(candidates[0]).Invoke(t, true, "unregisterCandidate", candidates[0].(neotest.SingleSigner).Account().PublicKey().Bytes())
+	e.CheckTxNotificationEvent(t, tx, 0, state.NotificationEvent{
```

Comment:
It's wrong API, CommitteeChange event is emitted during edge block OnPersist script execution. Ref. comment above for right API.